src/_posts/2020-12-04-bootstrap-5-beta-1.md

```diff
@@ -0,0 +1,150 @@
+
+There are some really awesome new features—RTL!—that have been added in this release, and they've come with some important changes. Keep reading for the lowdown and let us know what you think!
+
+## RTL
+
+![]()
+
+Our biggest addition to the project in years, we've finally added RTL support to Bootstrap! Please join me in giving @ffoodd—one of our newest contributors and author of the RTL pull request—a massive thank you. [The RTL pull request](https://github.com/twbs/bootstrap/pull/30980) includes nearly 50 cross-references to existing issues and PRs that have tried to implement the feature into our core.
+
+At a high level, our RTL approach includes a handful of changes:
+
+- **New RTL versions of our CSS dist file**, which includes our grid, Reboot, utilities, and standard bundles. [See the Contents page for a full list of files.](https://getbootstrap.com/docs/5.0/getting-started/contents/)
+
+- **[New RTL documentation](https://getbootstrap.com/docs/5.0/getting-started/rtl/)** to help you get started.
+
+- **[Five new RTL Examples](https://getbootstrap.com/docs/5.0/examples/#rtl)** that show our new RTL CSS in action, converting our Album, Checkout, Carousel, Blog, and Dashboard examples into their RTL equivalents.
+
+- **Two new cheatsheet kitchen sink pages**—the [default cheatsheet](https://getbootstrap.com/docs/5.0/examples/cheatsheet/) for our standard CSS and the [RTL cheatsheet](https://getbootstrap.com/docs/5.0/examples/cheatsheet-rtl/).
+
+Our approach is built on [RTLCSS](https://rtlcss.com/), an awesome project that helps reprocess an existing LTR stylesheet for RTL. We've classified it as an experimental feature for now, anticipating that we'll get some of this wrong. We're looking to the community to help us round out the feature as we wrap up some remaining todos.
+
+[This issue](https://github.com/twbs/bootstrap/issues/32330) has some outstanding todos already planned for our team to address. If you see something else we can improve upon, please consider opening an issue or pull request.
+
+[Read up on RTL in Bootstrap in our docs](https://getbootstrap.com/docs/5.0/getting-started/rtl/), including required HTML changes, a starter template, details on our approach, and more.
+
+## Renamed classes for logical properties
+
+Part of our approach to adding RTL to Bootstrap was to do so in a way that felt future-friendly to ourselves and the web at large. As such, we've embraced the spirit of CSS logical properties in some class names. It's a risky change because of the size and impact of the change, but we hope you'll appreciate it.
+
+Most of you have already interacted with logical properties thanks to our flex utilities—they replace direction properties like `left` and `right` in favor `start` and `end`. This makes the class names and values appropriate for LTR and RTL without any additional overhead.
+
+For example, in a LTR context, instead of `.ml-3` for `margin-left`, use `.ms-3`.
+
+## Popper.js v2
+
+...
```

Comment:
These should be the breaking Popper changes:

  - Removed `offset` option from our Tooltip/Popover and Dropdown plugins; this can still be achieved using the `popperConfig` parameter.
  - The `fallbackPlacement` option has become `fallbackPlacements`.